public/src/game.js

```diff
@@ -0,0 +1,46 @@
+var Game = function(frame) {
+  this.throwList = [];
+  this.frameList = [
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(true),
+  ];
+};
+
+Game.prototype.runningTotal = function() {
+  var trackThrow = 0;
```

Comment:
trackThrow is having a nice time being added to but I'm not sure if it's a team player.